### DISCUSSION TITLE
When will proxy mode support  GDI & GLYPH_SUPPORT_CACHE ？

### DISCUSSION THREAD

**Discussion by Mr-LZG** [FreeRDP/FreeRDP#10347] [General]
When will proxy mode support  GDI & GLYPH_SUPPORT_CACHE ？

**Comment by akallabeth** [comment]
when windows XP gets support again.
Feel free to create a pull request to support ancient systems.

note: ~~the mode works with FreeRDP 3 as we do not interpret data and just forward, so~~ I assume you´re talking about FreeRDP 2
